angular/src/lib/card/card.component.ts

```diff
@@ -8,9 +8,39 @@ import { Component, ElementRef, Input } from '@angular/core';
   styles: [],
   host: {
     'class': 'md-card',
+    '[class.md-card-clickable]': 'clickable',
+    '[attr.tabindex]': 'clickable ? "0" : null',
```

Comment:
Should the 0 have double quotes?

Reply:
Well you can't put more single quotes inside the single quotes .. I copied this syntax from alert-meeting.

Reply:
Gotcha - didn't know if it should be more like button where it's passed in as a number instead of string:
```
   '[attr.tabindex]': 'index === focusIndex ? 0 : -1',
```
Appears it doesn't matter.